api/client/proto/authservice.pb.go

```diff
@@ -1,5 +1,5 @@
 // Code generated by protoc-gen-gogo. DO NOT EDIT.
```

Comment:
So with this change the .pb.go file no longer lives next to the corresponding .proto file?

Reply:
Right. It Is The Way. I copied the `proto/` and `gen/proto/go` layout directly from Buf, in the hopes of making us standard looking.

Having protos beside generated files tends to get weird when you need multiple language clients. If we were starting from scratch today, I would go as far as suggest having the protos in an entirely separate repo, and `git submodule`-ing it like we do with others.